app/src/main/res/layout/empty_view_layout.xml

```diff
@@ -0,0 +1,29 @@
+<?xml version="1.0" encoding="utf-8"?>
+<LinearLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    android:layout_width="match_parent"
+    android:layout_height="match_parent"
+    android:orientation="vertical"
+    android:gravity="center">
+
+    <TextView
+        android:id="@+id/txtvtitle"

```

Comment:
Sorry for complaining so much. Could you please rename the ids? Something like `emptyViewTitle` and `emptyViewMessage` would be more specific. Additionally, it reduces the risk of having the same id twice on the same screen.